docs/tutorials/generate-custom-test-data.rst

```diff
@@ -0,0 +1,120 @@
+.. _ksql_generate-custom-test-data:
+
+Generate custom test data by using the ksql-datagen tool
+========================================================
+
+KSQL enables generating test data that complies with a custom schema
+that you define. 
+
+To generate test data, create an Apache Avro schema and pass it to the
+command-line tool, ``ksql-datagen``, which generates random data according

```

Comment:
suggest making this two sentences:

```
To generate test data, create an Apache Avro schema and pass it to 
the command-line tool, ``ksql-datagen``. This will generate random
data according to the schema you provide.
```